docs/source/compilation/index.rst

```diff
@@ -0,0 +1,23 @@
+.. _compilation:
+
+Compilation
+=============
+
+Before starting Indico compilation, this guide assumes you've previously
+:ref:`cloned the development version <cloning>`.
+The first step is to generate a local distribution archive. Navigate to the Indico source folder
+(by default, it is ``~/dev/indico/src``) and run the following commands::
+
+    ./bin/maintenance/build-wheel.py --target-dir dist indico --add-version-suffix
+
+
+This will generate a wheel distribution in the ``dist`` folder::
+
+    dist/
+      indico-2.3.dev0-py2-none-any.whl
```

Comment:
maybe use a filename that contains the suffix here? ;)

Reply:
Well, I didn't because this one looks simpler. But sure I can include it.